### PR TITLE
Release 2.21.0-rc.1

### PR DESCRIPTION
**Pull request by roidelapluie**
Signed-off-by: Julien Pivotto <roidelapluie@inuits.eu>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,7 +22,12 @@ validation.
 * [ENHANCEMENT] PromQL: Reduce the amount of data queried by remote read when a subquery has an offset. #7667
 * [ENHANCEMENT] Promtool: Add `--time` option to query instant command. #7829
 * [ENHANCEMENT] UI: Move "remove graph" closer to query area. #5990
```
Comment (brian-brazil) [inline CHANGELOG.md:24]: This sounds like a minor UI tweak, which we don't list.

**Review by brian-brazil — APPROVED** [review]

**Comment by roidelapluie** [comment]
I have rebased it so the tests reflect the current status of the 2.21 branch

**Comment by roidelapluie** [comment]
I want to have a few more hours to see if #7873 is easily actionnable before releasing this.

**Pull request merged**